config/externalnamenottested.go

```diff
@@ -590,4 +590,115 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 	//
 	// aws_imagebuilder_components resources can be imported by using the Amazon Resource Name (ARN)
 	"aws_imagebuilder_component": config.IdentifierFromProvider,
+
+	// accessanalyzer
+	//
+	// AccessAnalyzer ArchiveRule can be imported using the analyzer_name/rule_name
+	"aws_accessanalyzer_archive_rule": config.TemplatedStringAsIdentifier("", "{{ .parameters.analyzer_name }}/{{ .parameters.rule_name }}"),
+
+	// acmpca
+	//
+	// No import
+	// TODO: For now API is not normalized. While testing resource we can check the actual ID and normalize the API.
+	"aws_acmpca_permission": config.IdentifierFromProvider,
+	// aws_acmpca_policy can be imported using the resource_arn value
+	// Example: arn:aws:acm-pca:us-east-1:123456789012:certificate-authority/12345678-1234-1234-1234-123456789012
+	"aws_acmpca_policy": config.IdentifierFromProvider,
+
+	// appconfig
+	//
+	// AppConfig Extensions can be imported using their extension ID
+	"aws_appconfig_extension": config.IdentifierFromProvider,
+	// AppConfig Extension Associations can be imported using their extension association ID
+	"aws_appconfig_extension_association": config.IdentifierFromProvider,
+
+	// applicationinsights
+	//
+	// ApplicationInsights Applications can be imported using the resource_group_name
+	"aws_applicationinsights_application": config.ParameterAsIdentifier("resource_group_name"),
+
+	// apprunner
+	//
+	// App Runner Observability Configuration can be imported by using the arn
+	// Example: arn:aws:apprunner:us-east-1:1234567890:observabilityconfiguration/example/1/d75bc7ea55b71e724fe5c23452fe22a1
+	"aws_apprunner_observability_configuration": config.IdentifierFromProvider,
+	// App Runner VPC Ingress Connection can be imported by using the arn
+	// Example: arn:aws:apprunner:us-west-2:837424938642:vpcingressconnection/example/b379f86381d74825832c2e82080342fa
+	"aws_apprunner_vpc_ingress_connection": config.IdentifierFromProvider,
```

Comment:
Similar to the discussion above, but this time we do not have an annoying unknown component after the connection name. So, `name` looks like to be a naming field and we have a generated part (the last component). So, we could use (from the above example ARN) `example/b379f86381d74825832c2e82080342fa` as the external-name and treat `name` (Terraform configuration) argument as a naming field (i.e., remove it from `spec.forProvider`). 

One similar example is here:
https://github.com/upbound/provider-aws/blob/4194064bf89b91bae2f4739fab7d1245485b0980/config/externalname.go#L1527
for the `aws_cloudformation_stack` resource. But in this case, looks like we just normalized the external-name but still kept the naming argument `spec.forProvider.name`.

TLDR; we may at least normalize the external-name here by following a similar approach with `aws_cloudformation_stack`'s configuration. But we had better also have a TODO so that with some further effort, we can also normalize the `spec.forProvider` API.